features/step_definitions/navigation_steps.rb

```diff
@@ -230,6 +230,13 @@
   step %(I follow "#{$client_fullhostname}") if target == "sle-client"
 end
 
+When(/^I check the "(.*?)" client$/) do |target|
```

Comment:
@meaksh  use here this function, 

https://github.com/SUSE/spacewalk-testsuite-base/blob/d3392f6518166c10aafdb4560b6861c268d4cecb/features/step_definitions/command_steps.rb#L14

```node.run("hostname -f")``` 
we should add to the object node.hostname, but imho is better then the one here